### DISCUSSION TITLE
Kind of Building / key:building= value

### DISCUSSION THREAD

**Discussion by areynoua** [streetcomplete/StreetComplete#4337] [Q&A] [answered]
General question: should a building kind be determined by the current use or by architectural aspect / intent at construction time.

Specific case: here in Brussels there is a lot of single-family house from the '50s (something like 2 rooms per floor on 4 levels) that are now independent appartements. For those building, should we answer "single family house" or flats?

Thanks

**Comment by Helium314** [comment, marked as answer]
According to https://wiki.openstreetmap.org/wiki/Key:building it should be more the original type, but actual tagging is often more close to current use...
If the building received structural adjustments for the new use, e.g. separate staircase instead of passing though other people's rooms, I'd rather classify it as `apartments`. But that's not something you can usually see from outside, so it's up to you.

Personally for such buildings I tag `house` if there is a doorbell for one place, and `apartments` if there are 3 or more. If there are 2 bells with different names I try to judge from other things...

But you can always use the more generic `residential` answer when you're not sure.

**Comment by westnordost** [comment]
StreetComplete does not have the ability to tag both the current usage (`building:use`) and the original construction (`building`). (The former is only tagged if these two differ). If you want to tag both, you need to leave a note or use another editor.
